GitUI/Translation/English.xlf

```diff
@@ -9023,17 +9023,28 @@ branch names, tag names)</source>
   </file>
   <file datatype="plaintext" original="RepoObjectsTree" source-language="en">
     <body>
-      <trans-unit id="_searchTooltip.Text">
-        <source>Search</source>
+      <trans-unit id="_filterForSelected.Text">
+        <source>&amp;Filter for selected</source>
+        <target />
+      </trans-unit>
+      <trans-unit id="_filterForSelectedRefsMenuItem.Text">
+        <source>&amp;Filter for selected</source>
         <target />
       </trans-unit>
-      <trans-unit id="_showBranchOnly.Text">
-        <source>Filter the revision grid to show this branch only
-To show all branches, right click the revision grid, select 'view' and then the 'show all branches'</source>
+      <trans-unit id="_filterForSelectedRefsMenuItem.ToolTipText">
+        <source>Filter the revision grid to show selected (underlined) refs (branches and tags) only.
+Hold Ctrl while clicking to de/select multiple and include descendant tree nodes by additionally holding Alt.
```

Comment:
It's worth double checking if we use "CTRL", "ALT", etc., in other places to ensure consistency. I _think_ it is customary to write these keys in the upper case.

Reply:
I checked and `Ctrl` it hasn't been used in any translations before, but `ALT` has and it seems to be written all-caps to avoid confusion with the seemingly common Turkish word `alt`. So I'll apply that pattern and use all-caps.